src/schedule/ucc_schedule.h

```diff
@@ -138,10 +169,15 @@ static inline ucc_status_t ucc_task_complete(ucc_coll_task_t *task)
         status = task->super.status;
     }
 
+    if (task->executor && (task->flags & UCC_COLL_TASK_FLAG_EXECUTOR_STOP)) {
+        status = ucc_ee_executor_stop(task->executor);
```

Comment:
with this ucc_task_complete will return status returned from ucc_ee_executor_stop.  is that ok?

Reply:
agree, changed to return task status